pyproject.toml

```diff
@@ -50,22 +50,22 @@ Issues = "https://github.com/OpenAssetIO/OpenAssetIO/issues"
 # PyPI. Instead, boil down the first couple of sections to give a brief
 # overview.
 text = """
-    In modern creative pipelines, data is often managed by an authoritative system (Asset\
-    Management System, Digital Asset Manager, MAM, et. al).
+In modern creative pipelines, data is often managed by an authoritative system (Asset\
```

Comment:
Just noticed - it needs a space before the `\` to avoid the words running together.

Reply:
Spot. [f1bda67](https://github.com/OpenAssetIO/OpenAssetIO/pull/693/commits/f1bda6758da876a7ce291f3ec5ab6c7aedbd4f0b)

Reply:
They all need updating, not just this one.

Reply:
[44b6dca](https://github.com/OpenAssetIO/OpenAssetIO/pull/693/commits/44b6dca989533aa80287267bfbf02d53125bb95d)